stages/org.osbuild.ostree.deploy

```diff
@@ -151,14 +155,18 @@ def ostree_commit_deploy(tree, inputs, osname, remote, ref, kopts):
                *kargs, sysroot=tree, os=osname)
 
 
-def ostree_container_deploy(tree, inputs, osname, remote, kopts):
+def ostree_container_deploy(tree, inputs, osname, remote, verify_container, kopts):
     images = containers.parse_containers_input(inputs)
     for image in images.values():
         with containers.container_source(image) as (image_name, image_source):
             extra_args = []
             imgref = f"ostree-unverified-image:{image_source}"
 
-            target_imgref = 'ostree-unverified-registry'
+            if verify_container:
```

Comment:
renaming this var to indicate the target may also make things more clear.. but we want to keep the name kind of succinct.. Maybe:

```suggestion
            if target_gpgcheck:
```